binding.gyp

```diff
@@ -59,6 +59,34 @@
 					},
 					"libraries": [ '<(SOLETTA_LIBS)' ],
 					"dependencies": [ "collectbindings" ]
+				},
+				{
+					"target_name" : "generatecythonbindings",
```

Comment:
I was just following what they did... but I'm not sure those lines are being used.  I'll double check.